raiden/transfer/mediated_transfer/target.py

```diff
@@ -243,13 +241,7 @@ def handle_unlock(
             transfer.payment_identifier, transfer.lock.secrethash
         )
 
-        send_processed = SendProcessed(
-            recipient=balance_proof_sender,
-            message_identifier=state_change.message_identifier,
-            canonical_identifier=CANONICAL_IDENTIFIER_UNORDERED_QUEUE,
-        )
-
-        events.extend([payment_received_success, unlock_success, send_processed])
+        events.extend([payment_received_success, unlock_success])
```

Comment:
Why don't we send the `Processed` here any more?

Reply:
that is what the PR fix, only the Delivered is necessary.

Reply:
this seems to be a valid bug. Unlock needs to be answered by a `Processed`.